src/index.html

```diff
@@ -7,12 +7,71 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin
```

Comment:
Consider fixing the lint error 
<img width="706" alt="Screenshot 2024-06-23 at 14 38 47" src="https://github.com/mate-academy/layout_product-cards/assets/135232001/d4987144-f995-4a09-87fb-c8919f13d201">

Reply:
Thanks, done!